### DISCUSSION TITLE
Why this over sidebase?

### DISCUSSION THREAD

**Discussion by targetlucked69** [Hebilicious/authjs-nuxt#5] [Q&A] [answered]
There is an existing module - https://github.com/sidebase/nuxt-auth. Why should we use this instead of that?

Also with Authjs - https://github.com/wobsoriano/authey

**Comment by Hebilicious** [comment, marked as answer]
I've created this module because the existing solution weren't meeting my needs

Sidebase/nuxt-auth is 

1) Based on Next Auth and not on Auth.js
2) Reliant on node APIs and therefore wasn't compatible with edge platforms such as cloudflare workers.

Regarding Authey, it's not a Nuxt module. 

I would recommend to use this module if : 

- You're using Nuxt
- You want to use Auth.js (not next-auth)
- You want to have a solution that works on Node and Non-node runtimes

 But you could try to use all of them and see what works best for your use case.
